### DISCUSSION TITLE
Can we set cookie expiration times by role/user?

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.framework#2716] [Q&A]
I was just wondering if it was possible or worth noting as an enhancement to set cookie expiration dates based on User and/or Roles such as Host, Each Site Administrator, Authenticated User...?

For example if you want Host cookies to expire at different set amount of times from administrators or any other user or role you want set to override the default cookie time?  Host has one cookie time, while Administrators have another set cookie time and so on.  May need a switch to allow Host to let administrators manage cookie time permissions for the site they are administrating?

Again maybe we can something like this and I have not come across it.

Seems like it would be a security enhancement if not as I have not been through the Frameworks system for a while so not 100% what has all been added/modified yet.

**Comment by sbwalker** [comment]
Oqtane uses the standard .NET Core Identity library for setting the auth cookie.
